scenes/ui/game_menu/game_menu.gd

```diff
@@ -0,0 +1,31 @@
+extends Control
+
+@onready var quit_button := $Panel/VBoxContainer/MarginContainer/QuitButton
+
+@onready var panel = $Panel
+
+func _ready():
+	quit_button.pressed.connect(_on_quit_button_pressed)
+
+func _input(event):
+	if Input.is_key_pressed(KEY_ESCAPE):
```

Comment:
Using the KEY_ESCAPE button directly gives the behavior to flash the menu when you hold the "ESC" keyboard button.

I think it's safer to add a new input map: project->Input Map-> Add New Action -> j_toggle_game_menu

Check the bag and equiment as an example:
https://github.com/jonathaneeckhout/jdungeon/blob/main/scenes/player/equipment/Equipment.gd#L34

This will prevent the flashing